src/main/java/org/mycore/mir/metsprinter/MCRMetsPDFStreamingResource.java

```diff
@@ -131,16 +130,13 @@ public Response streamPDF(@PathParam("derivate") String derivate, @QueryParam("p
 
         MCRContent metsContent = getMetsContent(derivate);
 
-        String stylesheet = MCRConfiguration2.getString(MIR_PDF_XSLSTYLESHEET)
-            .orElseThrow(() -> MCRConfiguration2.createConfigurationException(MIR_PDF_XSLSTYLESHEET));
-
         MCRSession session = MCRSessionMgr.getCurrentSession();
         session.put("derivateID", derivate);
         if (pages != null) {
             session.put("ranges", pages);
         }
 
-        MCRXSLTransformer xslTransformer = new MCRXSLTransformer(stylesheet);
+		MCRXSLTransformer xslTransformer = (MCRXSLTransformer) MCRContentTransformerFactory.getTransformer("mets-print");
 
         byte[] byteArray;
         try(ByteArrayOutputStream out = new ByteArrayOutputStream() ) {
```

Comment:
The change should trigger a error in the  line `             xslTransformer.transform(metsContent, out, new MCRParameterCollector(false));` which can be fixed by checking if it is a MCRParameterizedTransformer:
```
if(xslTransformer instanceof MCRParameterizedTransformer paramTransformer) {
 paramTransformer.transform(metsContent, out, new MCRParameterCollector(false));
} else {
 xslTransformer.transform(metsContent, out);
}

This will allow to any transformer and not only XSL-Transformers
```